src/screenshotservice.cpp

```diff
@@ -174,12 +176,15 @@ ScreenshotResult *ScreenshotService::saveScreenshot(const QString &path)
         return result;
     }
 
-    QImage grab(compositor->grabWindow());
+    auto grabResult = compositor->contentItem()->grabToImage();
```

Comment:
Could prefer the type written explicitly for code readability/understandability. Was first wondering why the qsharedpointer include when there's nothing needing it.